_pages/homepage.md

```diff
@@ -53,93 +53,144 @@ flow:
-            more details.
-      - format: text
-        style: bg-white p-4 my-3
-        text_content:
-          text: |-
-            ## Firmware
+            [Arm Development Studio Morello Edition](https://developer.arm.com/architectures/cpu-architecture/a-profile/morello/development-tools#arm-development-studio) provides a development environment for bare-metal configurations and includes a "hello world" example.
 
-            The Development Platform is supported by Morello aware ports of standard open source firmware
-            components: [SCP firmware](https://github.com/ARM-software/SCP-firmware), [Trusted Firmware TF-A](https://www.trustedfirmware.org/), [UEFI EDK2](https://github.com/tianocore/edk2)
-      - format: text
-        style: bg-white p-4 my-3
-        text_content:
-          text: |-
-            ## Morello Linux Kernel
+            ## Android Enablement
 
-            Modified ACK (Android common Kernel).
-      - format: text
-        style: bg-white p-4 my-3
-        text_content:
-          text: |-
-            ## Nano Android
+            An evolving Android environment has been available for Morello since the first release in October 2020. 
 
-            An integrated Android stack supporting a reduced nano profile is hosted in the Morello code repositories.
+            This includes a minimal (nano) headless system Android (64bit) profile suitable for use with the FVP. Full Android boot will be supported on the Morello hardware platform
+
+            Support for Pure capability (Pure-cap) applications (along with several example ports) is provided by a Morello ACK (Android Common Kernel) and Bionic library variants built using the CHERI LLVM/Clang toolchain.
+
+            For more information on the status of the Android environment, please refer to `android-readme.rst` in the Documentation repository
+
+            Further information on the timeline for ongoing work in the Morello Kernel and the use of libshim in C libraries is available below.
+
+            ## Linux Enablement (2022 onwards)
+
+            Support for a Morello Linux environment is still under development. 
+
+            ### Initial Proof of Concept Linux environment: Expected end of 2021 (on FVP)
+            * Based around CHERI LLVM/Clang toolchain & Morello ACK.  Provides a prototype (shimmed) musl libC port and hello world userspace example integrated into a standard (A64) rootfs. 
+            * This provides a starting point for incremental enhancements targeting both the Morello hardware and FVP platforms.
+
+            ### Examples of planned work packages, 2022 onwards:
+
+            * Initial Linux environment support for hardware platform (and support for native compilation)
+            * Further evolution of musl lib C support & Pure-cap userspace application ports
```

Comment:
libC (consistent with the spelling above).

Reply:
Ack.